npm/@fastly/cli/package-lock.json

```diff
@@ -0,0 +1,552 @@
+{
+  "name": "@fastly/cli",
+  "version": "10.12.3",
+  "lockfileVersion": 3,
+  "requires": true,
+  "packages": {
+    "": {
+      "name": "@fastly/cli",
+      "version": "10.12.3",
+      "license": "Apache-2.0",
+      "bin": {
+        "fastly": "fastly.js"
+      },
+      "devDependencies": {
+        "decompress": "^4.2.1",
+        "decompress-targz": "^4.1.1"
+      },
+      "engines": {
+        "node": ">=16"
+      },
+      "optionalDependencies": {
+        "@fastly/cli-linux-386": "10.12.3",
+        "@fastly/cli-linux-arm64": "10.12.3",
+        "@fastly/cli-linux-x64": "10.12.3",
+        "@fastly/cli-win32-386": "10.12.3",
+        "@fastly/cli-win32-arm64": "10.12.3"
```

Comment:
I would suggest removing these lines from this file too:

```suggestion
```

Reply:
Good catch, thanks